tests/sentry/grouping/snapshots/test_fingerprinting/test_event_hash_variant/fingerprint_exception_type_and_sdk.pysnap

```diff
@@ -0,0 +1,38 @@
+---
+created: '2024-01-10T00:09:15.538545Z'
+creator: sentry
+source: tests/sentry/grouping/test_fingerprinting.py
+---
+config:
+  rules:
+  - attributes: {}
+    fingerprint:
+    - chunkloaderror
+    matchers:
+    - - sdk
+      - sentry.javascript.nextjs
+    - - type
+      - ChunkLoadError
+  version: 1
+fingerprint:
+- chunkloaderror
+title: 'ChunkLoadError: ChunkLoadError something something...'
+variants:
+  app:
+    component:
+      contributes: false
+      hint: custom fingerprint takes precedence
```

Comment:
In the future, would this say something like "internal fingerprint takes precedence"?